File: snapshop-android-dev/snapshop-android-dev/core/src/main/java/com/sg/core/param/ChangePassParam.kt
package com.sg.core.param
import android.os.Parcelable

import kotlinx.android.parcel.Parcelize

import androidx.annotation.Keep

import com.google.gson.annotations.SerializedName



@Parcelize
data class ChangePassParam(
    var userid: Int? = 0,
    var oldpass: String? = "",
    var newpassword: String? = ""
) : Parcelable

File: ClothesAPI/ClothesManament/Models/ChangePassParam.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ClothesManament.Models
{
    public class ChangePassParam
    {
        public int userid { set; get; }
        public string oldpass { set; get; }
        public string newpassword { set; get; }
    }
}